website/docs/docs/build/environment-variables.md

```diff
@@ -128,6 +128,15 @@ For example, suppose you'd like to run a full-refresh job in an XL warehouse, bu
 
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/Environment Variables/warehouse-override.png" title="Adding environment variables to your connection credentials"/>
 
+:::info Environment Variables and Snowflake OAuth Limitations
+Env vars work just fine with username/password and keypair (and thus in scheduled jobs), because it simply inserts jinja into the autogenerated `profiles.yml` which dbt Core then consumes by resolving the jinja to do an `env_var` lookup. 
+
+Env vars can be used in the Connection settings with Snowflake OAuth, but there are limitations. 
+
+Snowflake OAuth Connection fails to connect when supplied with an environment variable in the account/host field. 
```

Comment:
```suggestion
Something to note, if you supply an environment variable in the account/host field, Snowflake OAuth Connection will **fail** to connect. This happens because the field does not pass through jinja rendering, so dbt Cloud simply passes the literal `env_var `code into a URL string like `{{ env_var("DBT_ACCOUNT_HOST_NAME") }}.snowflakecomputing.com`, which is an invalid hostname.
```